### PR TITLE
fix logic for hypothesis testing

### PR DESCRIPTION
**Pull request by MikeLippincott**
Fix the logic for the hypothesis testing -> results unchanged.

### CODE REVIEW

File: 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py [hunk 1]
```diff
@@ -200,6 +194,43 @@
...
+    df = output_dfs[data_shuffle_df]
+    # split the feature column
+    df[
+        [
+            "feature_group",
+            "measurement",
+            "bone",
+            "parameter1",
+            "parameter2",
+            "parameter3",
```
Comment (jenna-tomkinson) [inline 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py:208]: Are these parameters defined somewhere? Currently, the naming is very vague. Note, I am coming from a very minimal background of the project so maybe this was already defined somewhere else.

Comment (jenna-tomkinson) [reply 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py:208]: Are these CellProfiler features maybe?

File: 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py [hunk 2]
```diff
@@ -217,21 +248,44 @@
...
+# In[13]:
+
+
+final_enrichment_half_baked_df.head(60)
+
+
+# In[14]:
+
+
+final_enrichment_df.head(60)
```
Comment (jenna-tomkinson) [inline 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py:277]: Why 60 for the head? Might consider reducing this if not necessary to make the notebook more simple.

File: 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py
Comment (jenna-tomkinson) [inline 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py:1]: Consider including markdown cells with titles for the code cells for readability. Might make it easier to follow.

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I didn't see anything glaring in the code, but I am a few steps behind on knowledge for the project. Most of my comments are questions and recommendations. Exciting to see this project almost complete!

File: 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py (hunk 1 above)
Comment (MikeLippincott) [reply 6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py:208]: Yes, and I do not use the parameters, they are just place holders

**Pull request merged**